### DISCUSSION TITLE
[Bug] Command 'tts' not found, but there are 15 similar ones.

### DISCUSSION THREAD

**Discussion by GiovZa** [coqui-ai/TTS#1962] [General Q&A]
### Describe the bug

Error message:

Command 'tts' not found, but there are 15 similar ones.

The tts command is not working for me after using a fresh Ubuntu 20.04 install. I tried using 18.04 but I got the same problem with more python installs that I thought was the problem so I swapped to 20.04 since it has more up-to-date python stuff. Here is everything I did in the terminal after creating a fresh VM:

### To Reproduce

sudo apt update

sudo apt install git

git clone https://github.com/coqui-ai/TTS

cd TTS

sudo apt install python3-pip

pip install click==8.0

pip install regex==2021.8.3

pip install -e .

tts --list_models

### Expected behavior

lists all models available

### Environment

```shell
- OS (Linux 18.04: Bionic Beaver)
- TTS (0.8.0)
```

**Comment by erogol** [comment]
It is hard to say something since it is not a reproducible problem. I think it is something particular about your system. If it'd be a common issue, we could see it in the CI tests. 

I move this to the discussions.

**Reply by GiovZa** [reply]
Fair enough. I'll try over the weekend to set this up on my laptop and/or a Pi and see if I can do the installation or not. Thank you.

**Reply by lonelyIslanders** [reply]
I also encountered the same problem, has anyone solved it?

**Reply by vamsi3296** [reply]
> I also encountered the same problem, has anyone solved it?
Create a venv/virtualenv and run everything in it. After that tts --list_models works.
[https://docs.python.org/3/library/venv.html]

**Comment by Ataullha** [comment]
I believe that, 
- creating a `virtual environment` can be a good solution
- another option that you can consider is using  `anaconda` or similar alternative and then try those steps again from the anaconda CLI can solve that. 
By using anaconda, you can ensure that the necessary packages are installed correctly, and that any potential conflicts are avoided.

In my experience, both of these approaches have worked well for me on my Linux system.

**Reply by dengzi118** [reply]
use pip install TTS this  situation can also occur   so who can tell me the right way  to install TTS

**Comment by Distantoff** [comment]
I went to the /home/user/.local/bin directory and launched from there "./tts"

**Reply by Distantoff** [reply]
Therefore, this path must be added to the PATH environment variable
